apps/progs.pl

```diff
@@ -131,6 +131,7 @@
     "aes-128-cbc", "aes-128-ecb",
     "aes-192-cbc", "aes-192-ecb",
     "aes-256-cbc", "aes-256-ecb",
+    "aria",
```

Comment:
ARIA is only available via the EVP interface, not as a command name.
Modes are supported.